context.go

```diff
@@ -76,9 +76,7 @@ func getTChannelParams(ctx context.Context) *tchannelCtxParams {
 
 // NewContext returns a new root context used to make TChannel requests.
 func NewContext(timeout time.Duration) (context.Context, context.CancelFunc) {
-	return NewContextBuilder(timeout).
-		setSpan(NewRootSpan()).
```

Comment:
@kriskowal OpenTracing cannot impose any specific restrictions on how a "trace identity" is represented in a string, because it's an implementation detail of the trace and they vary greatly. However, it is reasonable to require that `span.String()` returns _something_ that can be used to locate the trace in the respective UI. It's just that this "something" will be implementation specific. For example, Jaeger spans return a string like this: `73754768284bbd0f:73754768284bbd0f:0:0`. Note that the string will be unique for each span, but the first segment is a common trace id (implementation detail).